css/index.css

```diff
@@ -143,4 +144,80 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+.logo{
+    margin-top: 25px;
+    margin-left: 20px;
+}
+nav{
+    max-width:;
+    overflow: auto ;
+    overflow: hidden;
+
+}
+li{
+float: left;  
+}
+ul{
+    width:100%;
+    display:block;
+    margin-left: auto;
+    margin-right: auto;
+    width:50%;
+}
+li a{
+    width:100px;
+    display:block;
+    padding: 25px 25px;
+    
+}
+.jumbo img{
+    display:block;
+    margin-left: auto;
+    margin-right:auto;
+    width: 50%;
+    margin-bottom: 20px;
+}
+.gallery{
+    display:flex;
+   flex-wrap: wrap;
+ justify-content: space-between;
+}
+
```

Comment:
This part is correct and how it should be done, by only overriding the styles that change in the child (the color).  You can modify the CSS above with all the boxes and change a little bit in the index.html and it will successfully inherit all the parent styles.